examples/cucumber/README.md

```diff
@@ -0,0 +1,32 @@
+# saucectl espresso & cucumber example
+
+Example running saucectl with espresso & cucumber.
+
+## What You'll Need
+
+The steps below illustrate one of the quickest ways to get set up. If you'd like a more in-depth guide, please check out
+our [documentation](https://docs.saucelabs.com/testrunner-toolkit/installation).
+
+### Install `saucectl`
+
+```shell
+npm install -g saucectl
```

Comment:
@valentyn88 lets avoid set up using `npm` for espresso. Android users dont need `npm` locally to run this. Can you change this to the `curl` installation?

Reply:
Sure